packages/client/src/App.tsx

```diff
@@ -65,6 +65,7 @@ import { ViewRecord } from '@client/views/ViewRecord/ViewRecord'
 import { UserAudit } from './views/UserAudit/UserAudit'
 import { AdvancedSearchResult } from '@client/views/AdvancedSearch/AdvancedSearchResult'
 import { RegistrationList } from '@client/views/Performance/RegistrationsList'
+import { ListOfOrganizations } from '@client/views/Organization/ListOfOrganizations'
```

Comment:
Let's use the British spelling "Organisation" instead of "Organization" throughout the code so it's aligned with what's in the design.